schematics/cloud/authentication/emails/files/emailsCdkConstruct.ts.template

```diff
@@ -30,7 +30,8 @@ export class EmailsCdkConstruct extends Construct {
       {
         entry: path.join(__dirname, "./emailsLambda.ts"),
         environment: {
-          emailTemplatesBucketName: emailTemplatesBucket.bucketName,
+          EMAIL_TEMPLATES_BUCKET_NAME: emailTemplatesBucket.bucketName,
+          WEB_APP_URL: process.env.WEB_APP_URL || "https://janush.app",
```

Comment:
Is it possible not to hardcode `https://janush.app` here?

Reply:
It's only a boilerplate. I think it's better to leave it as it is and then customize it in the project